addons/binding/org.openhab.binding.onkyo/README.md

```diff
@@ -0,0 +1,78 @@
+# Onkyo Binding
+
+This binding integrates the Onkyo AV receivers.
+
+## Introduction
+Binding should be compatible with Onkyo AV receivers which support ISCP (Integra Serial Control Protocol) over Ethernet (eISCP).
+
+## Supported Things
+This binding supports only one thing: The Onkyo AV Receiver
+
+
+## Discovery
+This binding can discover the supported Onkyo AV Receivers. At the moment only the following models are supported:
+
+* TX-NR535
```

Comment:
I have only tested it with the TX-NR535. Probably other models will work as well since the communication part of this binding is based on the OH1 Onkyo binding.